.vscode/settings.json

```diff
@@ -1,7 +1,11 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "prettier.prettierPath": "./node_modules/prettier",
+  "editor.formatOnPaste": false,
+  "editor.formatOnType": false,
   "editor.formatOnSave": true,
+  "editor.formatOnSaveMode": "file",
+  "files.autoSave": "onFocusChange",
```

Comment:
I think we don't need this line... saving should be intentionally fired manually

what you think @luizstacio  ??

I use cmd+shift+s when I want to save all files

Reply:
Hm, but in which way this can change the `ctrl+z/ctrl+shift+z`? I didn't get it 🤔

Reply:
it helps identifying what files you changed last, then use undo/redo if needed...
if applying this change everything will be always saved when change file right ?

Reply:
also many times we need to change a lot of files to get a change done... if autoSave is enabled, live reload will break app you're running a lot of times...

saving should be manual, if want can execute `cmd+shift+s` to save all files, that's what I do when complete changes in multiple files

Reply:
I always used this personal setting on my VSCode and I never have problems with this. Some extensions recommend this type of autosave:
![CleanShot 2023-01-26 at 22 58 04](https://user-images.githubusercontent.com/2029172/214994181-7fa4ffbe-0a5e-44fc-a107-3f611bde3c86.png)

This is good to ensure that, if something VSCode you don't lose what you did I guess also.
About the live reloading, I don't think this is a point.

But idk, is a point of view, I can remove it, no problem.

Reply:
Let's see whats the rest think about @tomiiide @luizstacio

Reply:
we can always add later if @FuelLabs/frontend think it's good.. I personally don't like it

can wait for other to answer 

or just remove for now and add later in other PR, then we can quickly approve this one

Reply:
thing with `ctrl+z / ctrl+shift+z` is that if the file is unsaved, you know how many `undo` you need to get to the previous saved(valid) one

Reply:
ask for re-review when this is solved

Reply:
sounds good @pedronauck

Reply:
I also don't like auto-save, we probably should remove it from the project itself.

Reply:
removed